uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/payment/PaymentFactory.kt

```diff
@@ -12,30 +12,27 @@ class PaymentFactory {
 
     //TODO Implemente changes for guest checkout
     companion object {
-        fun createPresenter(provider: Provider?, view: BookingPaymentMVP.View): PaymentDropInMVP.Presenter? {
+        /*fun createPresenter(provider: Provider?, view: BookingPaymentMVP.View): PaymentDropInMVP.Presenter? {
```

Comment:
Do we need to remove it?

Reply:
I'm moving this creation logic to the Presenter